cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -566,7 +564,7 @@ public Catalog getTimeSeriesCatalog(String page, int pageSize, CatalogRequestPar
                         .units(row.get(AV_CWMS_TS_ID.AV_CWMS_TS_ID.UNIT_ID))
                         .interval(row.get(AV_CWMS_TS_ID.AV_CWMS_TS_ID.INTERVAL_ID))
                         .intervalOffset(row.get(AV_CWMS_TS_ID.AV_CWMS_TS_ID.INTERVAL_UTC_OFFSET));
-                if (this.getDbVersion() > Dao.CWMS_21_1_1) {
+                if (dbVersion > Dao.CWMS_21_1_1) {
```

Comment:
This clause can get removed, all dbs are now newer versions according to confluence doc.

Reply:
True, but we should actually formalize when that happens a bit more. Totally okay in this case but kinda of have the same issue in OpenDCS, can't keep code around to support everything forever without a rather high maintenance burden.

Reply:
I'll merge this and then remove the check in a new pr.